esmvaltool/recipes/recipe_climwip_brunner20esd.yml

```diff
@@ -0,0 +1,244 @@
+# ESMValTool
+# recipe_climwip.yml
+---
+documentation:
+  description: EUCP ClimWIP
```

Comment:
Could you expand the description a little bit?

Reply:
Will do!